### DISCUSSION TITLE
bug: map layer BUTTON visibility

### DISCUSSION THREAD

**Discussion by CyhnYldz** [acikkaynak/deprem-yardim-frontend#1148] [Ideas]
## Bug Definition
map layer button not visable enough

-- Please add discord contact information
** discord username: @afetharita#0001 **

## Bug environment
Describe the environment produces the bug. For example:
rc.afetharita.com

## Describe how you are producing the bug step by step

1. Go to '.RC..'rc.afetharita.com
2. Zoom in 
3. Bug appears

## Expected Behaviour
A clear and short text to decribe the expected behaviour.

## Screen shots
If possible, add screenshots to describe your bug.
![HaritaLayerBolumu](https://user-images.githubusercontent.com/111140617/220213725-4ba9f0d3-2343-4e6a-8257-95b7fde22c51.jpg)
## Desktop Information

- Operating System: [for example iOS]
- Browser [for example, safari]
- Version [for example 22]

## Mobile Phone Information
- Devıce: [for example iPhone6]
- Operating System (with the version): [for example iOS8.1]
- Version [for example default browser, safari]
- Browser Version [for example 22]

## Additional Context
Add any other context about the bug here

**Comment by nazdezar** [comment]
I would make this a low priority for now. We can review the icons and styling after the release.

**Comment by usirin** [comment]
@nazdezar agreed, gonna convert this to a discussion so we don't lose it.
